content/collections/advanced-techniques/en/sticky-bucketing.md

```diff
@@ -45,7 +45,7 @@ Remember that if you are targeting a dynamic cohort that users can flow in and o
 #### Enable sticky bucketing when:
 
 * You want to **give the user a consistent experience**, even if the user property you’re targeting changes. For example, if you’re running an experiment only in the United States, enabling sticky bucketing would ensure your users would see the same variant if they happened to travel outside the country.
-* You want to **decrease the percentage rollout** of an experiment (rolled out to half your users, for example) where the treatment group is not performing well. But you **don’t** want users to be moved from either the treatment or the control to the group that never saw either variant. Enabling sticky bucketing will keep users in their assigned groups even after you change the percentage rollout.
+* You want to **decrease the percentage rollout** of an experiment (rolled out to half your users, for example) where the treatment group is not performing well. But you **don’t** want users to be moved from either the treatment or the control to the group that never saw either variant (AKA 'off'). Enabling sticky bucketing keeps users in their assigned groups even after you change the percentage rollout. Increasing or decreasing percentage rollout doesn't cause users to move between treatment and control.
```

Comment:
@akhil-prakash What is 'off' in this context?

Reply:
off is for people not included in the experiment

Reply:
https://amplitude.com/docs/feature-experiment/workflow/feature-flag-rollouts#create-a-new-flag